scripts/download_model.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+MODEL_DIR=models
+DATA_DIR=data
+REPO_DIR=audioset_tagging_cnn
+
+# === Check for dirs first and get errors out of the way ===
+if [ -d "$MODEL_DIR" ]; then
+  echo $MODEL_DIR ' exists. Please delete it to continue'
+  exit 1
+fi
+
+# ============ Download the model ============
```

Comment:
remove the comments because they're in `echo`

Reply:
done